kotlin/openID4VP/src/main/java/io/mosip/openID4VP/common/FieldDeserializer.kt

```diff
@@ -0,0 +1,102 @@
+package io.mosip.openID4VP.common
+
+import kotlinx.serialization.DeserializationStrategy
+import kotlinx.serialization.SerializationException
+import kotlinx.serialization.builtins.ListSerializer
+import kotlinx.serialization.builtins.serializer
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.JsonNull
+import kotlinx.serialization.json.JsonObject
+import kotlinx.serialization.json.JsonPrimitive
+import kotlinx.serialization.json.boolean
+
+class FieldDeserializer(
```

Comment:
why in some places deserializer is different class, in some places it is inside dto ?